broker/server.js

```diff
@@ -58,6 +57,25 @@ app.get(generalEndpoints, (req, res) => {
     });
 });
 
+app.get("/Patient*", (req, res) => {
```

Comment:
To avoid making server.js a giant monolithic file that is impossible to support, please move this method in a new directory/file named broker/endpoints/Patient.js. You will have to export the method in that file.

Then replace the code in server..js with:

const PatientEndpoint = require("./endpoints/Patient.js");
app.get("/Patient*", PatientEndpoint);

Reply:
Done